<launch>

  <node ns="indoor/gvg/" name="localizerGVGNode" pkg="localizer" type="localizerGVGNode"  output="screen" args=" ">
    <!--<remap from="/odom" to="/indoor/odom"/>-->
    <remap from="/indoor/odom" to="/indoor/gvg/odom_combined"/>
    <param name="Wvv" value="0.075" type="double" />
    <param name="Wvw" value="0.00" type="double" />
    <param name="Www" value="0.0075" type="double" />
  </node>

</launch>
